### PR TITLE
ref: Refactor `photodiodeGhostSpot` logic

### LINKED ISSUE #331: CSS Variable for the photodiode color
- [ ] Flash between visible and invisible
- [ ] Color of the circle itself is set as a CSS variable in `App.css`

### LINKED ISSUE #402: Rename `startCodeTrial` as `beepTrial`
- [ ] Need a separate trial for setting up the photodiode (not `holdUpMarkerTrial`
- [ ] Remove the "Setting up" text and photodiode logic from `startCodeTrial`
- [ ] Rename as `beepTrial`

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- `photodiode-spot` flashes from visible -> hidden instead of black -> white
- `pdSpotEncode` uses `document.getElementId` instead of jquery
- Removes the jquery dependency
- Removes the "Setting up..." logic from `startCodeTrial`
- Renames the beep trial as `beepTrial` in `beep.js`
- Renames `startCodeTrial` as `initPhotodiodeTrial` in `initPhotodiode.js`

### CODE REVIEW

File: src/lib/markup/trials.css [hunk 1]
```diff
@@ -1,28 +1,28 @@
 /* Custom styling for the photodiode box. */
```
Comment (RobertGemmaJr) [inline src/lib/markup/trials.css]:
```suggestion
/* Custom styling for the photodiode box */
```

**Comment by eldu** [comment]
So this is the PR that has the `CONTINUE_ANYWAY` changes in the files changed but that's in another PR. #434 

Maybe since that change got merged in, you can merge `v3.3.1` into this one?

**Comment by RobertGemmaJr** [comment]
> Maybe since that change got merged in, you can merge `v3.3.1` into this one?

Oh whoops yeah you're totally right. I just forgot to merge the changes in

**Review by eldu — APPROVED** [review]
merge party !

**Pull request merged**